pkg/parse/run.go

```diff
@@ -443,12 +466,19 @@ func UpdateConflictManagerStatus(ctx context.Context, conflictErrs []status.Mana
 
 	for conflictingManager, conflictErrors := range conflictingManagerErrors {
 		scope, name := declared.ManagerScopeAndName(conflictingManager)
-		if scope != declared.RootReconciler {
-			klog.Infof("No need to notify namespace reconciler for the conflict")
-			continue
-		}
-		if err := prependRootSyncRemediatorStatus(ctx, k8sClient, name, conflictErrors, defaultDenominator); err != nil {
-			klog.Warningf("failed to add the management conflict error to RootSync %s: %v", name, err)
+		if scope == declared.RootReconciler {
```

Comment:
This little refactor hopefully only changes readability, comments, and logging. Otherwise it's functionally the same.